gensim/sklearn_api/basemodel.py

```diff
@@ -11,7 +11,7 @@
 from abc import ABCMeta, abstractmethod
 
 
-class BaseSklearnWrapper(object):
+class BaseTransformer(object):
```

Comment:
With this refactoring... I'm again wondering, is this class even helpful? Its `set_params()` does less than the implementation that would be inherited from sklearn's own `BaseEstimator`. The other methods just serve to enforce that any subclass override certain methods – a rigor that is possibly useful, but that sklearn does not, itself, impose within its own superclass-hierarchy.